packages/core/src/web/bridges/mcp-app-adapter.ts

```diff
@@ -19,3 +21,78 @@ export const requestDisplayMode: Methods["requestDisplayMode"] = ({ mode }) => {
 
   throw new Error("Modal display mode is not accessible in MCP App.");
 };
+
+type PickContext<K extends readonly (keyof McpUiHostContext)[]> = {
+  [P in K[number]]: McpUiHostContext[P];
+};
+
+const createExternalStore = <
+  const Keys extends readonly (keyof McpUiHostContext)[],
```

Comment:
Why do you need a const keyword here? First time I see it used for a type generic